services-directions/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java

```diff
@@ -39,7 +39,7 @@ public static Builder builder() {
    * @since 4.4.0
    */
   @Nullable
-  public abstract String routeIndex();
+  public abstract Integer routeIndex();
```

Comment:
This is breaking SemVer and it seems out of scope of this PR.

Reply:
Good point!